license.sh

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/env bash
 
+# Account for different word splitting in zsh which doesn't split the words in
+# variables causing errors like e.g.:
+# unknown flag: --account team_kong --raw
+#
+# rel: https://zsh.sourceforge.io/Doc/Release/Options.html
+function unset_zsh_opts(){
+  if [[ "${SHELL}" =~ .*zsh$ && -n ${ZSH_NAME} ]]; then
+    unsetopt SH_WORD_SPLIT
+  fi
```

Comment:
Can we add this logic to the existing cleanup function instead? and then rename that from `cleanup_kong_license_vars` to just `cleanup`?